builders/container-apps-internal-registry-demo/startup-script.sh

```diff
@@ -140,10 +160,18 @@ echo "===== Executing the extend phase ====="
   -log-level debug \
   -app $CNB_APP_DIR
 
+lifecycleExitCode=$?
+if [ $lifecycleExitCode -ne 0 ]; then
+    echo "----- Build failed -----"
```

Comment:
Nit: I'm not sure that this "----- Build failed -----" header adds much. Please consider removing it as the next error line should be sufficient from my perspective.